test_expect_failure/scala_test_jvm_flags/Empty.scala

```diff
@@ -0,0 +1,3 @@
+package test_expect_failure.scala_test_jvm_flags
+
+class Empty
```

Comment:
shouldn't this be some kind of no-op test or something?

Reply:
👍 I'll expand this out into a trivial test.

Reply:
Fixed.